src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/response/ObjectionResponseDTO.java

```diff
@@ -30,6 +33,10 @@
     @JsonProperty("status")
     private ObjectionStatus status;
 
+    @JsonProperty("links")
+    @JsonUnwrapped
+    private Links links;
```

Comment:
The spec mentions Objection has a links field, as the responseDTO is created using MapStruct, this was the best I could think of without more change, any suggestions welcome